.github/workflows/website-gh-pages.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           node-version: 18.x
 
       - name: Install dependencies
-        run: node common/scripts/install-run-rush.js update
+        run: node common/scripts/install-run-rush.js update --max-install-attempts 3
```

Comment:
Should this be install instead of update too

Reply:
I didn't write the script, but probably yes.  `rush update` should typically only be run on a dev machine when creating a PR.  I will update in this PR.